iree/compiler/Dialect/HAL/Transforms/PackAllocations.cpp

```diff
@@ -312,9 +312,13 @@ class PackAllocationsPass
           // Allocate a new bin for this slice.
           bins.push_back({offset, {}});
           targetBin = &bins.back();
-          offset = IREE::Util::align(
-              loc, builder.createOrFold<arith::AddIOp>(loc, offset, sliceSize),
-              offsetAlignment, builder);
```

Comment:
There is another call to `IREE::Util::align` above. You should also be able to remove that code now that the `Util::AlignOp` is written.